src/index.html

```diff
@@ -7,6 +7,388 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo logo">
+        <a href="#home"class="logo__link">
+          <img
+            src="./images/logo.png"
+            alt="logo"
+            class="logo__img"
+          >
+        </a>
+      </div>
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+              >Laptops & Computers</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Photo</a>
+          </li>
```

Comment:
```suggestion
          <li class="nav__item">
            <a href="#" class="nav__link">Tablets</a>
          </li>

          <li class="nav__item">
            <a href="#" class="nav__link">Photo</a>
          </li>
```
add blank line between multiple sibling blocks